python/mxnet/context.py

```diff
@@ -212,6 +216,14 @@ def gpu(device_id=0):
     return Context('gpu', device_id)
 
 
+def num_gpus():
+    count = ctypes.c_int()
+    check_call(_LIB.MXGetGPUCount(ctypes.byref(count)))
+    return count.value
+
+def gpus():
```

Comment:
This doesn't need to be a core API.

Reply:
where do you suggest I should add this, or would you like me to remove this?

Reply:
please remove for now.